packages/graphql-yoga/src/plugins/resultProcessor/push.ts

```diff
@@ -9,9 +9,12 @@ export function processPushResult(
   result: ResultProcessorInput,
   fetchAPI: FetchAPI,
 ): Response {
+  const timeoutInSeconds = 12
+
   const headersInit = {
     'Content-Type': 'text/event-stream',
     Connection: 'keep-alive',
+    'Keep-Alive': `timeout=${timeoutInSeconds}`,
```

Comment:
Why are you providing a timeout here? The timeout is:

> An integer that is the time in seconds that the host will allow an idle connection to remain open before it is closed.

[_MDN_](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive)

So you're actually creating a race here, you tell the browser to close the connection after 12 seconds but issue a ping after 12 seconds.

```suggestion
```

Reply:
Ah, you're right! I don't even remember why I added that :D